sympy/physics/control/tests/test_lti.py

```diff
@@ -524,6 +567,55 @@ def test_Series_functions():
     assert not S3.is_biproper
 
 
+def test_MIMO_Series_functions():
+    tfm1 = TransferFunctionMatrix([[TF1, TF2, TF3], [-TF3, -TF2, TF1]])
+    tfm2 = TransferFunctionMatrix([[-TF1], [-TF2], [-TF3]])
+    tfm3 = TransferFunctionMatrix([[-TF1]])
+    tfm4 = TransferFunctionMatrix([[-TF2, -TF3], [-TF1, TF2]])
+    tfm5 = TransferFunctionMatrix([[TF2, -TF2], [-TF3, -TF2]])
+    tfm6 = TransferFunctionMatrix([[-TF3], [TF1]])
+    tfm7 = TransferFunctionMatrix([[TF1], [-TF2]])
+
+    assert tfm1*tfm2 + tfm6 == Parallel(Series(tfm2, tfm1), tfm6)
+    assert tfm1*tfm2 + tfm7 + tfm6 == Parallel(Series(tfm2, tfm1), tfm7, tfm6)
+    assert tfm1*tfm2 - tfm6 - tfm7 == Parallel(Series(tfm2, tfm1), -tfm6, -tfm7)
+    assert tfm4*tfm5 + (tfm4 - tfm5) == Parallel(Series(tfm5, tfm4), tfm4, -tfm5)
+    assert tfm4*-tfm6 + (-tfm4*tfm6) == Parallel(Series(-tfm6, tfm4), Series(tfm6, -tfm4))
+
+    raises(TypeError, lambda: tfm1*tfm2 + TF1)
+    raises(ValueError, lambda: tfm1*tfm2 + a0)
+    raises(ValueError, lambda: tfm4*tfm6 - (s - 1))
+    raises(ValueError, lambda: tfm4*-tfm6 - 8)
+    raises(ValueError, lambda: (-1 + p**5) + tfm1*tfm2)
+
+    # Shape criteria.
+
+    raises(TypeError, lambda: -tfm1*tfm2 + tfm4)
+    raises(TypeError, lambda: tfm1*tfm2 - tfm4 + tfm5)
+    raises(TypeError, lambda: tfm1*tfm2 - tfm4*tfm5)
+
+    assert tfm1*tfm2*-tfm3 == Series(-tfm3, tfm2, tfm1)
+    assert (tfm1*-tfm2)*tfm3 == Series(tfm3, -tfm2, tfm1)
+
+    # Multiplication of a Series object with a SISO TF not allowed.
+
+    raises(TypeError, lambda: tfm4*tfm5*TF1)
+    raises(ValueError, lambda: tfm4*tfm5*a1)
+    raises(ValueError, lambda: tfm4*-tfm5*(s - 2))
+    raises(ValueError, lambda: tfm5*tfm4*9)
+    raises(TypeError, lambda: (-p**3 + 1)*tfm5*tfm4)
+
+    # Transfer function matrix in the arguments.
+    assert Series(tfm2, tfm1, evaluate=True) == Series(tfm2, tfm1).doit() == \
+        TransferFunctionMatrix(((TransferFunction(-k**2*(a2*s + p)**2*(s**2 + 2*s*wn*zeta + wn**2)**2 + (-a2*p + s)*(a2*p - s)*(s**2 + 2*s*wn*zeta + wn**2)**2 - \
+            (a2*s + p)**2, (a2*s + p)**2*(s**2 + 2*s*wn*zeta + wn**2)**2, s),), (TransferFunction(k**2, 1, s),)))
+
+    # calling doit() once will not expand the internal Series and Parallel objects.
+    assert Series(-tfm3, -tfm2, tfm1, evaluate=True) == Series(-tfm3, -tfm2, tfm1).doit() == \
+        TransferFunctionMatrix(((TransferFunction(k**2*(a2*s + p)**2*(s**2 + 2*s*wn*zeta + wn**2)**2 + (a2*p - s)**2*(s**2 + 2*s*wn*zeta + wn**2)**2 + (a2*s + p)**2, \
+            (a2*s + p)**2*(s**2 + 2*s*wn*zeta + wn**2)**3, s),), (TransferFunction(-k**2*(a2*s + p)*(s**2 + 2*s*wn*zeta + wn**2), (a2*s + p)*(s**2 + 2*s*wn*zeta + wn**2)**2, s),)))
+
```

Comment:
Add one test with `Parallel` instance as the argument and call `.rewrite(TransferFunctionMatrix)`?